statsmodels/stats/sandwich_covariance.py

```diff
@@ -17,10 +17,10 @@
 pinv(x) scale pinv(x)   used currently in linear_model, with scale is
 1d (or diagonal matrix)
 (x'x)^(-1) x' scale x (x'x)^(-1),  scale in general is (nobs, nobs) so
-pretty large
-general formulas for scale in cluster case are in
-http://pubs.amstat.org/doi/abstract/10.1198/jbes.2010.07136 which also
-has the second version
+pretty large general formulas for scale in cluster case are in [4],
+which can be found (as of 2017-05-20) at 
+http://www.tandfonline.com/doi/abs/10.1198/jbes.2010.07136
```

Comment:
I think in cases like this it would be better to switch to proper referencing (econ journals)
"Cameron, Gelbach and Miller (2011)"

(links are included in my Zotero references when I create the bibliography with it, but they are not necessary, especially if they are not stable)